src/keys/bip39.rs

```diff
@@ -247,12 +277,24 @@ impl fmt::Debug for Passphrase {
 #[derive(Clone, Zeroize, ZeroizeOnDrop)]
 pub struct Seed([u8; 64]);
 
+impl Seed {
+    pub fn as_bytes(&self) -> &[u8; 64] {
```

Comment:
Maybe call this `inner`?

Reply:
For keys we use `fn to_bytes(&self) -> [u8; LEN]` or `fn as_slice(&self) -> &[u8]`. Wanted to make something similar. The naming is a bit inconsistent. Not sure this function is needed at all.

Reply:
I would definitely keep the function. I think that for transparent wrappers like this `inner` makes the most sense. My second pick would be just `bytes`. `to_bytes` implies that it consumes the value, but this is by ref.

Reply:
Renamed to `bytes`.